### PR TITLE
fix check 3.2.1.c

### PR DESCRIPTION
**Pull request by yoavAqua**
Its was I Yoav Rotem :) 
fix that won't except comments (start with#) and eq flag instead of string compare in flag: "..... = 0"

### CODE REVIEW

**Comment by yoavrotems** [comment]
Hey sorry my mistake, should be "grep ^net.ipv4.conf.all.accept_source_route /etc/sysctl.conf /etc/sysctl.d/*"

File: cfg/1.1.0/definitions.yaml [hunk 1]
```diff
@@ -4313,10 +4313,13 @@ groups:
             
     - id: 3.2.1.c
       description: "Ensure source routed packets are not accepted"
-      audit: "grep net.ipv4.conf.all.accept_source_route /etc/sysctl.conf /etc/sysctl.d/*"   
+      audit: "grep ^[^#]net.ipv4.conf.all.accept_source_route /etc/sysctl.conf /etc/sysctl.d/*"
```
Comment (lizrice) [inline cfg/1.1.0/definitions.yaml]: I don't get it? The command in the CIS spec is 

```
grep "net\.ipv4\.conf\.all\.accept_source_route" /etc/sysctl.conf /etc/sysctl.d/*
```

(Note that the # in the CIS spec is supposed to indicate the prompt, not a comment line.)

Comment (yoavrotems) [reply cfg/1.1.0/definitions.yaml]: Yes but we get false positive for stuff that comes built-in as a comment, like in this case the file /etc/sysctl.conf contains the comment:

> #net\.ipv4\.conf\.all\.accept_source_route = 0

which will pass the current test, so we just need to filter it out, like with " **^net...** ", and we know for sure it's not a comment but the real intended line we want to test

Comment (lizrice) [reply cfg/1.1.0/definitions.yaml]: I don't think this is quite right, at the moment this is looking for exactly one character that isn't # between the beginning of the line, and `net...`

```
vagrant@kube:~$ echo "#net" | grep "^[^#]net"
vagrant@kube:~$ echo "net" | grep "^[^#]net"
vagrant@kube:~$ echo "anet" | grep "^[^#]net"
anet
vagrant@kube:~$ echo " net" | grep "^[^#]net"
 net
vagrant@kube:~$ echo "  net" | grep "^[^#]net"
```

**Comment by yoavrotems** [comment]
then maybe we should do something like this? 
grep ^\s*net.ipv4.conf.all.accept_source_route /etc/sysctl.conf /etc/sysctl.d/*

**Comment by yoavrotems** [comment]
You are correct, we took it under consider in the update and will change here two.
https://github.com/aquasecurity/linux-bench/pull/25#issuecomment-533920272
What means we will add "^\s*"

**Comment by yoavrotems** [comment]
There are more tests that have the same problem, but if no one opened an issues I guess that it's not commented by default like here, and because we fixed it version 2.0.0 maybe we should leave here the tests as is?
Or maybe just change it here too?
wwyd @lizrice ?

**Pull request merged**